sdk/eventhub/azure-eventhub/samples/sync_samples/client_creation.py

```diff
@@ -25,12 +25,12 @@
 CONSUMER_GROUP = "$Default"
 
 
-def create_producer_client():
+def create_producer_client() -> None:
     print('Examples showing how to create producer client.')
 
     # Create producer client from connection string.
 
-    producer_client = EventHubProducerClient.from_connection_string(
+    producer_client: EventHubProducerClient = EventHubProducerClient.from_connection_string(
```

Comment:
don't think we need this annotated, since the method only returns this type. Same comment applies for everywhere EventHubProducerClient/EventHubConsumerClient/EventDataBatch are returned.